process/coordinator/process.go

```diff
@@ -588,7 +588,7 @@ func (tc *transactionCoordinator) CreateMbsAndProcessCrossShardTransactionsDstMe
 
 	headerHash, err := core.CalculateHash(tc.marshalizer, tc.hasher, hdr)
 	if err != nil {
-		return createMBDestMeExecutionInfo.miniBlocks, createMBDestMeExecutionInfo.numTxAdded, false, nil
+		return createMBDestMeExecutionInfo.miniBlocks, createMBDestMeExecutionInfo.numTxAdded, false, nil // TODO: check if we should return err here
```

Comment:
If we returns err here all the processing will be dropped. If we let it like this the processing will be stop here as we return false as allProcessed so the proposer can propose at least what it has been processed until now. It is hard to believe that this core.CalculateHash method will work with the first header but not for the second one. Anyway, if somehow we will have this case at least the proposer could propose something here without reverting everything.

Reply:
added comment + log.Warn print